index.html

```diff
@@ -91,82 +91,77 @@ <h2>My Recent Works
             </div>
         </div>
         <div class="portfolio-item">
             <img src="" alt="">
             <div class="portfolio-main">
-                <h3>Multi-Post Stories<br>Gain+Glory</h3>
-                <ul>
-                    <li>Ruby on rails</li>
-                    <li>css</li>
-                    <li>JavaScript</li>
-                    <li>html</li>
-                </ul>
-                <button onclick="location.href='#';">See Project</button>
+                <h3></h3>
+                <ul></ul>
+                <button onclick="revealDetails(3)">See Project</button>
             </div>
         </div>
         <div class="portfolio-item">
             <img src="" alt="">
             <div class="portfolio-main">
-                <h3>Multi-Post Stories<br>Gain+Glory</h3>
-                <ul>
-                    <li>Ruby on rails</li>
-                    <li>css</li>
-                    <li>JavaScript</li>
-                    <li>html</li>
-                </ul>
-                <button onclick="location.href='#';">See Project</button>
+                <h3></h3>
+                <ul></ul>
+                <button onclick="revealDetails(4)">See Project</button>
             </div>
         </div>
         <div class="portfolio-item">
             <img src="" alt="">
             <div class="portfolio-main">
-                <h3>Multi-Post Stories<br>Gain+Glory</h3>
-                <ul>
-                    <li>Ruby on rails</li>
-                    <li>css</li>
-                    <li>JavaScript</li>
-                    <li>html</li>
-                </ul>
-                <button onclick="location.href='#';">See Project</button>
+                <h3></h3>
+                <ul></ul>
+                <button onclick="revealDetails(5)">See Project</button>
             </div>
         </div>
     </section>
+    <section id="detailCont">
+        <div id="portfolio-details">
+            <img id="detailImg" src="" alt="">
+            <h1 id="detailH1"></h1>
+            <ul id="detailUl"></ul>
+            <p id="detailP"></p>
+            <div class="buttons">
+                <button>
+                    See Live
+                    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path fill-rule="evenodd" clip-rule="evenodd" d="M5 12C5 8.13401 8.13401 5 12 5C12.5523 5 13 4.55228 13 4C13 3.44772 12.5523 3 12 3C7.02944 3 3 7.02944 3 12C3 16.9706 7.02944 21 12 21C16.9706 21 21 16.9706 21 12C21 11.4477 20.5523 11 20 11C19.4477 11 19 11.4477 19 12C19 15.866 15.866 19 12 19C8.13401 19 5 15.866 5 12ZM16 3C15.4477 3 15 3.44772 15 4C15 4.55228 15.4477 5 16 5H17.5858L11.2929 11.2929C10.9024 11.6834 10.9024 12.3166 11.2929 12.7071C11.6834 13.0976 12.3166 13.0976 12.7071 12.7071L19 6.41421V8C19 8.55228 19.4477 9 20 9C20.5523 9 21 8.55228 21 8V4C21 3.44772 20.5523 3 20 3H16Z" fill="white"/>
```

Comment:
- [ ] Just like the previous reviewer suggested, there seems to be an overflow on the x-axis (left to right) on mobile devices as shown below 👇 Please fix this issue to achieve a more responsive design 📱 
![image](https://user-images.githubusercontent.com/46839548/211167950-b2cd9c19-6103-4a00-8410-d11c761c90a7.png)